enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -67,7 +68,7 @@ if (hasPremiumFeature("whitelabel")) {
         {
           key: "landing-page",
           display_name: t`Landing Page`,
-          type: "string",
+          widget: NormalizedTextSettingInput,
```

Comment:
I'm not sure if it's a good idea to have some kind of APIs that we could normalize each input. We do have `onBeforeChanged`
https://github.com/metabase/metabase/blob/dc56106904ba82a4575d1437d85e906b74f8fcfa/frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.jsx#L92-L97
and `onChanged`, but that doesn't do what I wanted

https://github.com/metabase/metabase/blob/dc56106904ba82a4575d1437d85e906b74f8fcfa/frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.jsx#L105-L110

I want a kind of API that compute the value before passing it to `onChange`. Maybe we could call it `normalize` or something, but I don't want to do that right now before discussing with other FE enginners.